### ISSUE TITLE
Backups are always full after upgrading to 18.04 / 0.40

### ISSUE THREAD

**Issue by haslinger** [mail-in-a-box/mailinabox#1508]
I upgraded mailinabox to 0.40 by switching to a new Ubuntu 18.04 vm. Restoring the backups and the other setup steps worked fine.
Now I'm seeing issues with all backups being full and the list of Available backups in the Backup Status
page of the web interface is empty. Backups are done via rsync to another 18.04 linux box.

Running `mailinabox/management/backup.py --status` returns with 
```
[]

Storage for unmatched files: 1732454293  
```

The backups are actually created, e.g. the last full backup looks like this:
```
drwxrwxr-x  2 stefan stefan 4096 Jan 21 03:03 duplicity-full-signatures.20190121T020021Z.sigtar.gpg
drwxrwxr-x  2 stefan stefan 4096 Jan 21 03:03 duplicity-full.20190121T020021Z.manifest.gpg
drwxrwxr-x  2 stefan stefan 4096 Jan 21 12:24 duplicity-full.20190121T112300Z.vol1.difftar.gpg
```

Any ideas, what I could have done wrong are highly appreciated.

**Comment by aroundmyroom** [comment]
I indeed also see only 4K files being transferred

drwxrwxrwx+  2 root root 4.0K Jan 21 21:45 duplicity-full.20190121T204400Z.vol10.difftar.gpg
drwxrwxrwx+  2 root root 4.0K Jan 21 21:46 duplicity-full.20190121T204400Z.vol11.difftar.gpg
drwxrwxrwx+  2 root root 4.0K Jan 21 21:47 duplicity-full.20190121T204400Z.vol12.difftar.gpg

problem also: config file of ssh is not being respected during the duplicity and rsync/ssh part where the initial config of the rsync part is using the config part (I use port 2345 and that one is working and accepted during the config in the UI, but during the backup -p 22 is used and the config file of ssh is not respected.

**Comment by haslinger** [comment]
@aroundmyroom you don't see 4K files, they are directories :-)

**Comment by alento-group** [comment]
@durd submitted a pull request on this issue here:
https://github.com/mail-in-a-box/mailinabox/pull/1511

**Comment by zoof** [comment]
Commenting out the line suggested in https://discourse.mailinabox.email/t/backup-via-rsync-seems-sort-of-broken/4345/28?u=zoof does not work for me.  It is making full backups every night and I am going to run out of space soon -- I'm willing to help in any way that I can.

**Comment by zoof** [comment]
Ok -- with the line commented out, it is now working as it is supposed to.  It was probably in the instruction that one should delete old backups and of course I hadn't...  After I deleted them, it works.

**Comment by haslinger** [comment]
Indeed, the PR in #1511 would fix this, but I'll keep this issue open, until it is merged into the code base.
